component/FloatingAction.js

```diff
@@ -396,6 +396,18 @@ class FloatingAction extends Component {
     }
     this.reset();
   }
+  
+  _updateState = (nextState, callback) => {
+    const { onStateChange } = this.props;
+    this.setState(nextState, () => {
+      if (typeof callback == 'function') {
+        callback();
+      }
+      if (typeof onStateChange == 'function') {
+        onStateChange(this.state);
```

Comment:
I'm not a big fan of sending the entire state... because the state is part of the component... will be better to send a new object with specific data... like **isActive**.